apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../../util/reconfiguredChai';
+import CodeReviewTimelineReview from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineReview';
+import {codeReviewTimelineElementType} from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineElement';
+import javalabMsg from '@cdo/javalab/locale';
+import Comment from '@cdo/apps/templates/instructions/codeReview/Comment';
+
+const DEFAULT_REVIEW = {
+  id: 1,
+  createdAt: '2022-03-31T04:58:42.000Z',
+  isClosed: false,
+  projectVersion: 'asdfjkl',
+  isVersionExpired: false
```

Comment:
This looks pretty great to me! Do you have a test case for isVersionExpired: true? Or is that unnecessary?

Reply:
I have the unit test `passes project version and is version expired to CodeReviewTimelineElement` which will check that the right prop is passed through

Reply:
What happens when isVersionExpired is true?

Reply:
I'll add another test!